community/kernel/src/main/java/org/neo4j/kernel/impl/util/ArrayQueueOutOfOrderSequence.java

```diff
@@ -33,6 +33,7 @@
     private long[] highestGapFreeMeta;
     private final SequenceArray outOfOrderQueue;
     private long[] metaArray;
+    private long highestEverSeen = 0;
```

Comment:
You should also initialize `highestEverSeen` in `set( long number, long[] meta )` and perhaps print it in `toString`